framework/include/cppmicroservices/Bundle.h

```diff
@@ -376,6 +376,26 @@ class US_Framework_EXPORT Bundle
    */
   std::string GetLocation() const;
 
+  /**
+    * Retrieves the resolved symbol from bundle shared library and returns a function pointer associated with it
+    * 
+    * @param1 Handle to the Bundle's shared library
+    * @param2 Name of the symbol
```

Comment:
for consistency with the other doxygen comments I would suggest using the ```@param <name> <description>``` form instead of ```@param1```, ```@param2```, etc...